addons/html_editor/static/src/others/embedded_components/plugins/excalidraw_plugin/excalidraw_dialog/excalidraw_dialog.js

```diff
@@ -31,13 +31,17 @@ export class ExcalidrawDialog extends Component {
     }
 
     checkInput() {
-        let potentialURL = this.inputRef.el.value;
+        this.state.hasError = false;
+        let potentialURL = this.inputRef.el.value?.trim();
         if (!potentialURL) {
+            this.state.isInsertionDisabled = true;
             return false;
         }
+        this.state.isInsertionDisabled = false;
         potentialURL = checkURL(potentialURL, excalidrawWebsiteDomainList);
         if (!potentialURL) {
             this.state.hasError = true;
+            this.state.isInsertionDisabled = true;
```

Comment:
Revert back to the previous code, but add the input value in the state:
```suggestion
        this.state.hasError = false;
        this.state.urlValue = this.inputRef.el?.value?.trim() || "";
        let potentialURL = this.state.urlValue;
        if (!potentialURL) {
            return false;
        }
        potentialURL = checkURL(potentialURL, excalidrawWebsiteDomainList);
        if (!potentialURL) {
            this.state.hasError = true;
```